NuRadioMC/EvtGen/generator.py

```diff
@@ -1279,60 +1296,56 @@ def generate_eventlist_cylinder(filename, n_events, Emin, Emax,
             # we need to be careful to not double cound events. electron CC interactions apear twice in the event list
             # because of the two distinct showers that get created. Because second interactions are only calculated
             # for mu and tau cc interactions, this is not a problem.
-            mask_tau_cc = (data_sets["interaction_type"] == 'cc') & (np.abs(data_sets["flavors"]) == 16)
-            mask_mu_cc = (data_sets["interaction_type"] == 'cc') & (np.abs(data_sets["flavors"]) == 14)
-            mask_leptons = mask_tau_cc | mask_mu_cc
+            mask_tracks = data_sets["interaction_type"] == 'cc' & np.abs(data_sets["flavors"]) != 12
```

Comment:
I'm against this change. You might save a line of code but it makes it more error prone in case we might have more initial particles in the future. It is always better to be explicit than implicit.

Reply:
Also here I have a hard time to see that we would want to use this function for other particles than neutrinos. But this is less explicit and I can change it again

Reply:
I reverted the changes here